client/components/swipeable/index.js

```diff
@@ -0,0 +1,212 @@
+import classnames from 'classnames';
+import { Children, useState, useEffect, useRef } from 'react';
+
+import './style.scss';
+
+export const Swipable = ( {
```

Comment:
Seems like we have a typo:
```suggestion
export const Swipeable = ( {
```